docs/content/en/docs/getting-started/_index.md

```diff
@@ -15,58 +25,266 @@ In your local environment the following tools are required for provisioning a K8
-> 
-> ```
-> W0128 11:24:54.792095  27657 warnings.go:70] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
-> ```
-> 
-> For more information, check out issue [#267](https://github.com/k8ssandra/k8ssandra/issues/267).
+For more information, check out issue [#267](https://github.com/k8ssandra/k8ssandra/issues/267).
+{{% /alert %}}
+
+## Verify your K8ssandra installation
+
+Depending upon your K8 configuration, initialization of your K8ssandra installation can take a few minutes. To check the status of your K8ssandra deployment, use the `kubectl` command:
+
+```bash
+kubectl get pods
+NAMESPACE     NAME                                                  READY   STATUS      RESTARTS   AGE
+default       k8ssandra-cass-operator-6666588dc5-rrbwx              1/1     Running     0          15m
+default       k8ssandra-dc1-default-sts-0                           2/2     Running     0          15m
+default       k8ssandra-dc1-stargate-7db4dbfdd5-dxk5v               1/1     Running     0          15m
+default       k8ssandra-grafana-b6f7978c4-cwdwg                     2/2     Running     0          15m
+default       k8ssandra-kube-prometheus-operator-5556885bd6-2tn4t   1/1     Running     0          15m
+default       k8ssandra-reaper-k8ssandra-64cc9d57d9-rc2z4           1/1     Running     0          12m
+default       k8ssandra-reaper-k8ssandra-schema-pxdcz               0/1     Completed   0          12m
+default       k8ssandra-reaper-operator-cc46fd5f4-hlmck             1/1     Running     0          15m
+default       prometheus-k8ssandra-kube-prometheus-prometheus-0     2/2     Running     1          15m
+```
 
-In later steps, you can upgrade your k8ssandra via `helm upgrade` commands, for example to access services from outside Kubernetes via a Traefik Ingress controller.
+NOTE: k8ssandra prefixes the pod names. Stargate takes 4 minutes to register as ready.
 
-## Defaults
 
-K8ssandra comes out of the box with a set of default values tailored to getting up and running quickly.  These defaults
-are intended to be a great starting point for smaller-scale local development and should be carefully customized for 
-production deployments.
 
-For more information on the configuration options available, and the developer focused defaults provided, take a look 
-at the [k8ssandra reference]({{< ref "/docs/reference/k8ssandra/" >}}) documentation.
+
+Once all the pods are in the `Running` or `Completed` state, you can check the health of your K8ssandra cluster.
+
+{{% alert title="Tip" color="success" %}}
+The pod `k8ssandra-reaper-k8ssandra-schema-xxxxx` runs once and does not persist.
+{{% /alert %}}
+
+The actual Cassandra node name from the listing above is `k8ssandra-dc1-default-sts-0`. If you've configured multiple nodes, `sts-0` will increment.
```

Comment:
The name is composed of multiple components. We may want to call those out:

- `k8ssandra` - cluster name
- `dc1` - datacenter name
- `default-sts` - rack name + `sts` (meaning stateful set)
- `0` - node id within the rack, note this value is ordinal and generated by the underlying stateful set